profiles/kentik_snmp/cisco/cisco-asr.yml

```diff
@@ -109,4 +109,21 @@ sysobjectid:
   - 1.3.6.1.4.1.9.1.2746  # Cisco C1121X-8PLTEP ISR
   - 1.3.6.1.4.1.9.1.2961  # Cisco C8200-1N-4T Edge Router
   - 1.3.6.1.4.1.9.6.1.23.3.13.0.4  # Cisco RV340 Dual WAN Gigabit VPN Router
-  
+
+metrics:
+  - MIB: CISCO-VOICE-DIAL-CONTROL-MIB
+    table:
+      name: cvCallVolIfTable
+      OID: 1.3.6.1.4.1.9.9.63.1.3.8.5
+    symbols:
+      - name: cvCallVolMediaIncomingCalls
+        OID: 1.3.6.1.4.1.9.9.63.1.3.8.5.1.1
+      - name: cvCallVolMediaOutgoingCalls
+        OID: 1.3.6.1.4.1.9.9.63.1.3.8.5.1.2
+    metric_tags:
+      - MIB: IF-MIB
+        table: ifTable
```

Comment:
these 2 lines are unnecessary in a `metric_tag` configuration. see [this sample](https://github.com/kentik/snmp-profiles/blob/2f640c2f8202c491e11bcdc916ccd5b07343d9e2/profiles/kentik_snmp/cisco/cisco-all-devices.yml#L68-L73) for an example of calling the `IF-MIB` from a table in the `CISCO-IF-EXTENSION-MIB`